library/kani_macros/src/lib.rs

```diff
@@ -89,17 +89,48 @@ pub fn derive_arbitrary(item: TokenStream) -> TokenStream {
     derive::expand_derive_arbitrary(item)
 }
 
+#[proc_macro_attribute]
+pub fn requires(attr: TokenStream, item: TokenStream) -> TokenStream {
+    attr_impl::requires(attr, item)
+}
+
+#[proc_macro_attribute]
+pub fn ensures(attr: TokenStream, item: TokenStream) -> TokenStream {
+    attr_impl::ensures(attr, item)
+}
```

Comment:
What about assigns? We might don't need it yet, so at least create a GitHub issue to track that we need to add it here.

Reply:
Has been created https://github.com/model-checking/kani/issues/2594